### PR TITLE
Update Set-AzureRmVMSqlServerExtension to remove infinite loop

### PR DESCRIPTION
**Pull request by michaelcohenmicrosoft**

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@michaelcohenmicrosoft__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (michco). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

File: src/ResourceManager/Compute/Commands.Compute.Test/Commands.Compute.Test.csproj [hunk 1]
```diff
@@ -243,6 +244,7 @@
     <None Include="ScenarioTests\RunnerTests.csv">
       <CopyToOutputDirectory>Always</CopyToOutputDirectory>
     </None>
+    <None Include="ScenarioTests\SqlIaaSExtensionTests.ps1" />
```
Comment (markcowl) [inline src/ResourceManager/Compute/Commands.Compute.Test/Commands.Compute.Test.csproj]: @michaelcohenmicrosoft You need to add this, and the recorded mock to the csproj, and both should be marked as 'CopyAlways' or 'CopyIfNewer'

**Comment by michaelcohenmicrosoft** [comment]
@hovsepm Tests have been added and are passing

**Comment by markcowl** [comment]
on demand run here: http://azuresdkci.cloudapp.net/view/1-AzurePowerShell/job/powershell-demand/699/

**Pull request merged**